### ISSUE TITLE
Attempting to download a workflow via Relay CLI, got "illegal base64 data at input byte 1573"

### ISSUE THREAD

**Issue by highb** [puppetlabs/relay#58]
## Describe the Bug
Trying to download the workflow with the Relay CLI and I got an error:
```
2020/08/27 12:08:15 the workflow body was in the wrong format. illegal base64 data at input byte 1573
```

## Expected Behavior
I can download the workflow with no errors.

## Steps to Reproduce
1. Create an example workflow via the Web UI
2. Run the workflow once
3. Attempt to download it via the Relay CLI
```
➜  relay git:(relay_poc) ✗ relay workflow list
+-------------+-----------------+
| NAME        | LAST RUN NUMBER |
+-------------+-----------------+
| onboarding  | 1               |
| offboarding | 0               |
+-------------+-----------------+
➜  relay git:(relay_poc) ✗ relay workflow download onboarding --debug
2020/08/27 12:09:12 GET /api/workflows/onboarding HTTP/1.1
Host: api.relay.sh
User-Agent: Go-http-client/1.1
Accept: application/vnd.puppet.relay.v20200615+json
Authorization: Bearer X
Accept-Encoding: gzip

2020/08/27 12:09:12 HTTP/2.0 200 OK
Content-Length: 1058
Access-Control-Allow-Origin: https://app.relay.sh
Alt-Svc: clear
Content-Security-Policy: default-src 'none'; base-uri 'none'
Content-Type: application/json
Date: Thu, 27 Aug 2020 19:09:12 GMT
Referrer-Policy: no-referrer
Relay-Request-Id: ea6561bc-c899-4332-9c14-0ea6d97bc23c
Strict-Transport-Security: max-age=31536000
Vary: Origin
Via: 1.1 google
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-Permitted-Cross-Domain-Policies: none
X-Xss-Protection: 0

{"access":{"permission_grants":[{"permission":{"name":"workflow.edit","description":"Edit workflows"},"granted":"all"},{"permission":{"name":"workflow.approve","description":"Approve workflow steps"},"granted":"all"},{"permission":{"name":"workflow.delete","description":"Delete workflows"},"granted":"all"},{"permission":{"name":"workflow.view","description":"View workflows"},"granted":"all"},{"permission":{"name":"workflow.run","description":"Run workflows"},"granted":"all"}]},"workflow":{"name":"onboarding","description":"","created_at":"2020-08-06T21:21:56.238205Z","updated_at":"2020-08-06T21:21:56.238205Z","latest_revision":{"id":"5868002a-44c6-42ea-8400-75b0c0d96bfa"},"most_recent_run":{"workflow":{"name":"onboarding"},"run_number":1,"revision":{"id":"5868002a-44c6-42ea-8400-75b0c0d96bfa"},"state":{"status":"success","started_at":"2020-08-27T18:33:11Z","ended_at":"2020-08-27T18:33:18Z"},"created_by":{"key":"","type":"user","user":{"id":"ebe287d5-a171-4d12-99cf-884438e043be","name":"Brandon High","email":"bhigh@opengov.com"}}},"state":{}}}
2020/08/27 12:09:12 GET /api/workflows/onboarding/revisions/5868002a-44c6-42ea-8400-75b0c0d96bfa HTTP/1.1
Host: api.relay.sh
User-Agent: Go-http-client/1.1
Accept: application/vnd.puppet.relay.v20200615+json
Authorization: Bearer X
Accept-Encoding: gzip

2020/08/27 12:09:12 HTTP/2.0 200 OK
Connection: close
Access-Control-Allow-Origin: https://app.relay.sh
Alt-Svc: clear
Content-Security-Policy: default-src 'none'; base-uri 'none'
Content-Type: application/json
Date: Thu, 27 Aug 2020 19:09:12 GMT
Referrer-Policy: no-referrer
Relay-Request-Id: 8b931fe5-58f7-4960-b684-fce7174398d7
Strict-Transport-Security: max-age=31536000
Vary: Origin
Via: 1.1 google
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-Permitted-Cross-Domain-Policies: none
X-Xss-Protection: 0

{"revision":{"id":"5868002a-44c6-42ea-8400-75b0c0d96bfa","parameters":{"message":{"default":"Hello world!","description":"The message to output from the step"}},"steps":[{"name":"hello-world","description":"","type":"container","image":"relaysh/core","spec":{"message":{"$type":"Parameter","name":"message"}},"input":["echo \"$(ni get -p {.message})\""],"references":{"parameters":[{"name":"message"}]}}],"triggers":[],"raw":"IyBUaGlzIGlzIGEgc2FtcGxlIHdvcmtmbG93IHRvIGhlbHAgeW91IGdldCBzdGFydGVkCiMgRnVsbCBndWlkZSBhdmFpbGFibGUgYXQgaHR0cHM6Ly9yZWxheS5zaC9kb2NzL2dldHRpbmctc3RhcnRlZAoKYXBpVmVyc2lvbjogdjEKZGVzY3JpcHRpb246IEEgc2ltcGxlIGhlbGxvIHdvcmxkIGV4YW1wbGUKIyBQYXJhbWV0ZXJzIGFsbG93IGRhdGEgdG8gYmUgcGFzc2VkIGluIHRvIGEgcnVuIGV4ZWN1dGlvbi4gU2VlIGh0dHBzOi8vcmVsYXkuc2gvZG9jcy9yZWZlcmVuY2UvcmVsYXktd29ya2Zsb3dzLyNwYXJhbWV0ZXJzCnBhcmFtZXRlcnM6CiAgbWVzc2FnZToKICAgIGRlc2NyaXB0aW9uOiBUaGUgbWVzc2FnZSB0byBvdXRwdXQgZnJvbSB0aGUgc3RlcAogICAgZGVmYXVsdDogSGVsbG8gd29ybGQhCiMgRW5hYmxlIGEgdHJpZ2dlciB0byBhdXRvbWF0ZSB3b3JrZmxvdyBleGVjdXRpb24KIyB0cmlnZ2VyczoKIwojIFNjaGVkdWxlIHRyaWdnZXJzIGFyZSBhdmFpbGFibGUgdG8gcnVuIHRoaXMgd29ya2Zsb3cgYXQgYSBzcGVjaWZpYyBpbnRlcnZhbC4gVW5jb21tZW50IHRvIHJ1biBob3VybHkKIyAtIG5hbWU6IHNjaGVkdWxlCiMgICBzb3VyY2U6CiMgICAgIHR5cGU6IHNjaGVkdWxlCiMgICAgIHNjaGVkdWxlOiAnKiAqICogKiAqJwojICAgYmluZGluZzoKIyAgICAgcGFyYW1ldGVyczoKIyAgICAgICBtZXNzYWdlOiAiT0sgY2FtcGVycywgcmlzZSBhbmQgc2hpbmUsIGFuZCBkb24ndCBmb3JnZXQgeW91ciBib290aWVzICdjYXVzZSBpdCdzIGNvb29vbGQgb3V0IHRoZXJlISIKIwojIEEgbGlicmFyeSBvZiBwcmUtY29uZmlndXJlZCB3ZWJob29rIHRyaWdnZXJzIGlzIGF2YWlsYWJsZSBmb3IgY29tbW9uIGludGVncmF0aW9ucy4gVGhpcyB3ZWJob29rIHdpbGwgdHJpZ2dlciBleGVjdXRpb24gdXBvbiBhIHBhZ2VyZHV0eSBpbmNpZGVudAojIFNlZSBodHRwczovL3JlbGF5LnNoL2ludGVncmF0aW9ucy8gZm9yIG1vcmUgYXZhaWxhYmxlIHRyaWdnZXJzCiMgLSBuYW1lOiBwYWdlcmR1dHkKIyAgIHNvdXJjZToKIyAgICAgdHlwZTogd2ViaG9vawojICAgICBpbWFnZTogcmVsYXlzaC9wYWdlcmR1dHktdHJpZ2dlci1pbmNpZGVudC10cmlnZ2VyZWQKIyAgIGJpbmRpbmc6CiMgICAgIHBhcmFtZXRlcnM6CiMgICAgICAgbWVzc2FnZTogIUZuLmNvbmNhdAojICAgICAgIC0gJ+KaoO+4jyAqPCcKIyAgICAgICAtICFEYXRhIGluY2lkZW50VVJMCiMgICAgICAgLSAnfCcKIyAgICAgICAtICFEYXRhIGluY2lkZW50VGl0bGUKIyAgICAgICAtICc+KiDimqDvuI8nCiMgICAgICAgLSAiXG5BZmZlY3RlZCBzZXJ2aWNlOiAiCiMgICAgICAgLSAhRGF0YSBzZXJ2aWNlTmFtZQojCiMgQSBiYXNpYyBzdGVwIGVjaG9pbmcgeW91ciBtZXNzYWdlLiBUcnkgcnVubmluZyB0aGVuIG5hdmlnYXRpbmcgdG8gdGhlIGxvZ3MgdGFiIHRvIHNlZSB0aGUgb3V0cHV0LgpzdGVwczoKLSBuYW1lOiBoZWxsby13b3JsZAogIGltYWdlOiByZWxheXNoL2NvcmUKICBzcGVjOgogICAgbWVzc2FnZTogIVBhcmFtZXRlciBtZXNzYWdlCiAgaW5wdXQ6CiAgLSBlY2hvICIkKG5pIGdldCAtcCB7Lm1lc3NhZ2V9KSIKIwojIEEgbGlicmFyeSBvZiBwcmUtY29uZmlndXJlZCBzdGVwcyBpcyBhdmFpbGFibGUgZm9yIGNvbW1vbiBpbnRlZ3JhdGlvbnMuIFRoaXMgc3RlcCB3aWxsIGZvcndhcmQgYSBtZXNzYWdlIHRvIHNsYWNrCiMgU2VlIGh0dHBzOi8vcmVsYXkuc2gvaW50ZWdyYXRpb25zLyBmb3IgbW9yZSBhdmFpbGFibGUgc3RlcHMKIyAtIG5hbWU6IHNlbmQtc2xhY2stbWVzc2FnZQojICAgaW1hZ2U6IHJlbGF5c2gvc2xhY2stc3RlcC1tZXNzYWdlLXNlbmQKIyAgIGRlcGVuZHNPbjoKIyAgIC0gaGVsbG8td29ybGQKIyAgIHNwZWM6CiMgICAgIGNvbm5lY3Rpb246ICFDb25uZWN0aW9uIHt0eXBlOiBzbGFjaywgbmFtZTogbXktc2xhY2stYWNjb3VudH0KIyAgICAgY2hhbm5lbDogI3JlbGF5LWFsZXJ0cwojICAgICB1c2VybmFtZTogUGFnZXJEdXR5IHZpYSBSZWxheQojICAgICBtZXNzYWdlOiAhUGFyYW1ldGVyIG1lc3NhZ2UK","created_at":"2020-08-27T18:33:02.08452Z"}}
2020/08/27 12:09:12 the workflow body was in the wrong format. illegal base64 data at input byte 1573
Error: An unknown error has occured. Rerun with --debug to troubleshoot.
  • illegal base64 data at input byte 1573

You have recieved an error in debug mode. If the error persists you may file a bug report at https://github.com/puppetlabs/relay/issues
```

## Environment
 - Version `relay version` and `relay --version` don't show the version but it looks like it is `v4.1.1`
```
➜  relay git:(relay_poc) ✗ ll /usr/local/bin/relay
Permissions Size User        Date Modified Name
lrwxr-xr-x    32 brandonhigh 27 Aug 12:03  /usr/local/bin/relay -> ../Cellar/relay/v4.1.1/bin/relay
```
 - Platform OSX 10.15.5

**Comment by rick-a-lane-ii** [comment]
This client is using [URLEncoding](https://github.com/puppetlabs/relay/blob/deb6777b76732d5b3e3942106a0cd6482bb08253/pkg/client/workflow.go#L247) and the API is using StdEncoding.  URLEncoding shouldn't be necessary here.

I've confirmed that updating this to StdEncoding fixes the issue.

**Comment by rick-a-lane-ii** [comment]
A fix has been applied and pushed to the master branch.
Official verification is pending release.

**Comment by rick-a-lane-ii** [comment]
This has been fixed in the latest Relay CLI release (v4.2.1).
